packages/babel-core/src/config/helpers/config-api.ts

```diff
@@ -47,7 +47,7 @@ export type PluginAPI = {
 export function makeConfigAPI<SideChannel extends Context.SimpleConfig>(
   cache: CacheConfigurator<SideChannel>,
 ): ConfigAPI {
-  const env: any = value =>
+  const env: any = (value: any) =>
```

Comment:
We can probably do `env: EnvFunction`?

Reply:
No, the method definition is too complicated for ts to automatically deduce.

Reply:
No, the method definition is too complicated for ts to automatically deduce.

Reply:
awesome! Thanks!